### ISSUE TITLE
convert footer to menu block

### ISSUE THREAD

**Issue by hcpss-banderson** [HCPSS/hcpss_schoolsite_theme#11]
I think we need to abstract out the menu contained in the footer of page.tpl.php into a menu and insert it with a block. Otherwise someone will have to manually update each one we deploy to the ES sites.

**Comment by hcpss-mbarger** [comment]
Definitely need to convert both the main navigation and footer into blocks.  I've been holding off of converting them to menus as I'd like some more flexibility around how those regions are designed.  Right now they are simple lists but I'd like them to be much richer in the future.

**Comment by hcpss-mbarger** [comment]
Here's the todo list for the block extraction:
- [x]  ARL
- [x]  Atholton HS
- [x] Centennial HS
- [x] Glenelg HS
- [x] Hammond HS
- [x] Homewood
- [x] Howard HS
- [x] Long Reach HS
- [x] Marriotts Ridge HS
- [x] Mt. Hebron HS
- [x] Oakland Mills HS
- [x] Reservoir HS
- [x] River Hill HS
- [x] Wilde Lake HS

**Comment by hcpss-banderson** [comment]
They also have to all be created for the ES/MS schools.

**Comment by hcpss-banderson** [comment]
Menus are not fieldable so your hands are kinda tied. I really can't tell, but this issue indicates that menu items are entities in D8:

https://www.drupal.org/node/916388

So I guess the answer for D7 is a content type/entity and views. An easy short term hack is to take all the markup as it is and dump it into a block if you need to buy some time before implementing a better solution in D7 or maybe D8.

**Comment by hcpss-banderson** [comment]
If menu items are entities in D8, they are still not fieldable, I checked. So, D8 is not going to solve this.

For the ES/MS schools, I guess the thing to do is deploy a generic set of menus that work for all schools along with the deployment module. But if I do that, and you create HS menu blocks by hand, then we will loose the ability to ever use the deployment module on the HS sites again because if we do, it will create another block like the one we created by hand and place it.

Maybe this is a way around that?

https://www.drupal.org/project/features_extra

Then we could export blocks as a feature.
